### PR TITLE
updating Deny virtual machine with un managed disk policy to Built-in

### PR DESCRIPTION
**Pull request by aarunraaj**
## Overview/Summary

Changed Deny virtual machine with Un Managed disk from custom policy to Built-in policy.

## Testing Evidence
image

image

### Testing URLs

The below URLs can be updated where the placeholders are, look for `{YOUR GITHUB BRANCH NAME HERE - Remove Curly Brackets Also}` & `{YOUR GITHUB BRANCH NAME HERE - Remove Curly Brackets Also}`, to allow you to test your portal deployment experience.

> Please also replace the curly brackets on the placeholders `{}`

#### Azure Public

[Deploy To Azure](https://portal.azure.com/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2Faarunraaj%2FEnterprise-Scale%2Fpolicy-refresh-unmanageddisk%2FeslzArm%2FeslzArm.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2Faarunraaj%2FEnterprise-Scale%2Fpolicy-refresh-unmanageddisk%2FeslzArm%2Feslz-portal.json)

#### Azure US Gov (Fairfax)
[Deploy To Azure](https://portal.azure.com/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2Faarunraaj%2FEnterprise-Scale%2Fpolicy-refresh-unmanageddisk%2FeslzArm%2FeslzArm.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2Faarunraaj%2FEnterprise-Scale%2Fpolicy-refresh-unmanageddisk%2FeslzArm%2Feslz-portal.json)

## As part of this Pull Request I have

- [ ] Checked for duplicate [Pull Requests](https://github.com/Azure/Enterprise-Scale/pulls)
- [ ] Associated it with relevant [issues](https://github.com/Azure/Enterprise-Scale/issues), for tracking and closure.
- [ ] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Enterprise-Scale/tree/main)
- [ ] Performed testing and provided evidence.
- [ ] Ensured [contribution guidance](https://github.com/Azure/Enterprise-Scale/wiki/ALZ-Contribution-Guide) is followed.
- [ ] Updated relevant and associated documentation.
- [ ] Updated the ["What's New?"](https://github.com/Azure/Enterprise-Scale/wiki/Whats-new) wiki page (located: `/docs/wiki/whats-new.md`)

### CODE REVIEW

**Comment by jtracey93** [comment]
@springstone, did this get done by you or does this still need doing?

**Comment by aarunraaj** [comment]
@jtracey93 , its done, but experienced some failure during deployment. So, running few tests.

File: docs/wiki/Whats-new.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ Here's what's changed in Enterprise Scale/Azure Landing Zones:
 - Added new initiative and assignment to enable Azure Compute Security Baseline compliance auditing for Windows and Linux virtual machines (`Enforce-ACSB`)
 - Updated the initiative `Deny-PublicPaaSEndpoints` to include additional policies available to block public access for PaaS services
   - Updated [storage](https://www.azadvertizer.net/azpolicyadvertizer/b2982f36-99f2-4db5-8eff-283140c09693.html) and [Key Vault](https://www.azadvertizer.net/azpolicyadvertizer/405c5871-3e91-4644-8a63-58e19d68ff5b.html) to use new policies using the `/publicNetworkAccess` alias
-- Added new policy and assignment to deny deployment of virtual machines using unmanaged OS disks (Deny-VM-with-unmanaged-disk)
+- Added new assignment to deny deployment of virtual machines and scalesets using unmanaged OS disks.
```
Comment (Springstone) [inline docs/wiki/Whats-new.md:99]:
```suggestion
- Added new assignment to deny deployment of virtual machines and virtual machine scale sets using unmanaged OS disks.
```

**Review by Springstone — CHANGES_REQUESTED** [review]
@aarunraaj Looks good, almost there. However, we also need to update the portal accelerator (remove Audit option) and eslzArm (remove parameters toplevelmanagementgroupprefix and policyEffect).

**Comment by aarunraaj** [comment]
> @aarunraaj Looks good, almost there. However, we also need to update the portal accelerator (remove Audit option) and eslzArm (remove parameters toplevelmanagementgroupprefix and policyEffect).

@Springstone, this is complete, and ran another successful test.

**Comment by Springstone** [comment]
@aarunraaj many thanks, can you kindly merge branch conflicts on this pull request.

**Review by Springstone — APPROVED** [review]
Looks good.

**Review by jtracey93 — APPROVED** [review]

**Pull request merged**